src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/processedenvelopes/ProcessedCcdType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.processedenvelopes;
+
+public enum ProcessedCcdType {
+    AUTO_ATTACHED_CASE,
```

Comment:
*to* case

Reply:
this not for action, actually case or exception

Reply:
I mean that we are not attaching a case. we are attaching something TO a case.

Reply:
but the type is case? affected type is case?

Reply:
I think Rafał treats this enum as describing CCD action type, while you're treating it as one describing the kind of CCD case. So maybe it should be one of the two or sth similar?
```
ATTACH_DOCUMENTS
CREATE_EXCEPTION_RECORD
```
or
```
SERVICE_CASE
EXCEPTION_RECORD
```
depending on how we look at it.

Reply:
I think this field just defining  what ID is referencing. ID represents - a case or exception record.